applications/MappingApplication/custom_modelers/mapping_geometries_modeler.h

```diff
@@ -7,12 +7,10 @@
 //  License:         BSD License
 //                   Kratos default license: kratos/license.txt
 //
+//  Main authors:    Philipp Bucher
```

Comment:
this one (header and source) was done by Peter, Tobi and me

Reply:
In a new PR